foursight_core/app_utils.py

```diff
@@ -65,6 +63,9 @@ class AppUtilsCore:
     LAMBDA_MAX_BODY_SIZE = 5500000  # 6Mb is the "real" threshold
 
     def __init__(self):
+        # dmichaels/2022-07-20/C4-826: Apply identity globally.
+        apply_identity_globally()
```

Comment:
Not currently going to work because of dependency in foursight-cgap/fourfront on getting ES_HOST environment variable. Added TODO comment to rework this in my branch (C4-826-indentity-ize-with-view-info).